src/libStd_helper/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2019 Fondazione Istituto Italiano di Tecnologia (IIT)
+# All Rights Reserved.
+# Authors: Giulio Romualdi <giulio.romualdi@iit.it>
+
+# set target name
+
+set(LIBRARY_TARGET_NAME Std_helper)
```

Comment:
Note that in general libraries get installed in the `/usr/local/lib` prefix, that it may be shared with a lot of other pages, so it is a good idea to avoid generic names such as `Std_helper`.

Reply:
You are right. What do you suggest? `Walking_controller_Std_helper`?

Reply:
Yes, prefixing with a project specific make sense, I do not love mixing camelcase and underscore, but that is a personal opinion.

Reply:
Done